shared/haml/course_explorer_table.haml

```diff
@@ -19,13 +19,10 @@
+- courses << { name: "CS Fundamentals: Accelerated Version", starts: 6, ends: 12, description: "Introductory course for early readers.", subtitle: "Condensed version of curriculum in one 20 hour course for older students", img: "/shared/images/courses/logo_tall_course1.jpg", color: "rgba(51, 224, 239, .7)", long: "Code.org’s CS Fundamentals courses blend online, self-guided, and self-paced tutorials with “unplugged” activities that require no computer at all. Students will create computer programs that will help them learn to collaborate with others, develop problem-solving skills, and persist through difficult tasks. They will study programming concepts, computational thinking, digital citizenship, and develop interactive games or stories they can share. Students at every grade level (including pre-readers) can begin CS Fundamentals with no prior programming experience." }
+- courses << { name: "CS Discoveries", starts: 7, ends: 9, description: "Introductory computer science course that be a semester or year long.", subtitle: "Semester or full year course", link: "/educate/csd", img: "/images/fit-500/csd/ms.jpg", color: "#1AC7D6", long: "CS Discoveries is an introductory Computer Science course that empowers students to create authentic artifacts and engage with CS as a medium for creativity, communication, problem solving, and fun. The course will be piloted in Spring 2017, with full rollout Summer 2017. No prerequisites needed. This course works for beginners or students with experience in CS Fundamentals." }
+- courses << { name: "CS Principles", starts: 9, ends: 12, description: "A year-long rigorous introductory computer science course at the high-school level covering the Internet, data, and programming. Can be run as a non-AP course.", subtitle: " Full year course", link: "/educate/csp", img: "/shared/images/teacher-announcement/csp-widgets.png", color: "#00adbc", long: "In fall 2016, the College Board launched its newest AP® course, AP Computer Science Principles. The course introduces students to the foundational concepts of computer science and challenges them to explore how computing and technology can impact the world. The AP Program designed AP Computer Science Principles with the goal of creating leaders in computer science fields and attracting and engaging those who are traditionally underrepresented with essential computing tools and multidisciplinary opportunities. The curriculum is flexible to be taught as a normal course or as an AP course. No prerequisites needed. This course works for beginners or students with experience in CS Fundamentals or CS Discoveries."}
```

Comment:
Is description used for anything? Or is it just long? Would rename long to description and get rid of the other.